src/tests/ftest/util/server_utils_params.py

```diff
@@ -500,11 +503,10 @@ def __init__(self, base_namespace, index, provider=None, max_storage_tiers=MAX_S
         for name in self._provider.split(";"):
             if name in self.REQUIRED_ENV_VARS:
                 default_env_vars.extend(self.REQUIRED_ENV_VARS[name])
+        if not self.using_dcpm:
+            default_env_vars.append("PMEMOBJ_CONF=sds.at_create=0")
```

Comment:
This cannot be done in `__init__()` because we've only set default parameters.   This needs to be done in `get_params()` where we set the values for the engines and storage under the section commented:
```
# Update the env vars with any missing or different required setting
```

Reply:
https://github.com/daos-stack/daos/pull/14371/files#diff-89439b64e4865a0fc2da96697625d9d0b1342fa4db06b450a352fb8d8d3dc24e

Reply:
Done